x-pack/legacy/plugins/lens/public/indexpattern_plugin/operation_definitions/terms.tsx

```diff
@@ -109,35 +109,8 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn> = {
       });
     orderOptions.push({
       value: toValue({ type: 'alphabetical' }),
-      text: i18n.translate('xpack.lens.indexPattern.terms.orderAlphabetical', {
-        defaultMessage: 'Alphabetical',
-      }),
+      text: 'Alphabetical',
```

Comment:
Looks like you got rid of the i18n here?

Reply:
Lost in copy-pasting